src/cr/cube/cube.py

```diff
@@ -415,13 +415,19 @@ def _ca_as_0th(self):
     def _cube_dict(self):
         """dict containing raw cube response, parsed from JSON payload."""
         cube_dict = copy.deepcopy(self._cube_response)
+        if self._cube_idx_arg:
+            cube_dict["cube_idx"] = self._cube_idx_arg
```

Comment:
Do we really need to set this specifically on the dictionary? I.e. don't we already have it in `self._cube_idx_arg`? And if so, why can't we use the `self._cube_idx_arg` in the other place (where we need to decide)?